wormchain/contracts/tools/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@wormhole-foundation/wormchain-contract-tools",
+  "version": "0.0.1",
+  "description": "scripts for working with wormchain contracts",
+  "main": "deploy_wormchain.ts",
+  "scripts": {
+    "deploy-wormchain": "ts-node deploy_wormchain.ts",
+    "test-wormchain": "ts-node test_wormchain.ts",
+    "deploy-and-test": "npm run deploy-wormchain && npm run test-wormchain"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
Why ISC?  AFAIU the rest of the repo is Apache-2

Reply:
removing